src/routes/docs/products/auth/multi-tenancy/+page.markdoc

```diff
@@ -0,0 +1,414 @@
+
+const client = new Client()
+    .setEndpoint('https://<REGION>.cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>');
+
+const teams = new Teams(client);
+
+// Invite a member to the tenant
+const membership = await teams.createMembership(
+    'example_corp',      // Team/tenant ID
+    ['admin'],           // Member's role in the tenant
+    'user@example.com',  // User's email
+    undefined,           // userId (optional)
+    undefined,           // phone (optional)
+    'https://example.com/accept-invite' // Redirect URL after accepting
+);
+```
+```client-flutter
+import 'package:appwrite/appwrite.dart';
+
+final client = Client()
+    .setEndpoint('https://<REGION>.cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>');
+
+final teams = Teams(client);
+
+// Invite a member to the tenant
+final membership = await teams.createMembership(
+    teamId: 'example_corp',   // Team/tenant ID
+    roles: ['admin'],         // Member's role in the tenant
+    email: 'user@example.com', // User's email
+    url: 'https://example.com/accept-invite' // Redirect URL after accepting
+);
+```
+```client-apple
+import Appwrite
+
+let client = Client()
+    .setEndpoint("https://<REGION>.cloud.appwrite.io/v1")
+    .setProject("<PROJECT_ID>")
+
+let teams = Teams(client)
+
+// Invite a member to the tenant
+let membership = try await teams.createMembership(
+    teamId: "example_corp",    // Team/tenant ID
+    roles: ["admin"],          // Member's role in the tenant
+    email: "user@example.com", // User's email
+    url: "https://example.com/accept-invite" // Redirect URL after accepting
+)
+```
+```client-android-kotlin
+import io.appwrite.Client
+import io.appwrite.services.Teams
+
+val client = Client(context)
+    .setEndpoint("https://<REGION>.cloud.appwrite.io/v1")
+    .setProject("<PROJECT_ID>")
+
+val teams = Teams(client)
+
+// Invite a member to the tenant
+val membership = teams.createMembership(
+    teamId = "example_corp",    // Team/tenant ID
+    roles = listOf("admin"),    // Member's role in the tenant
+    email = "user@example.com", // User's email
+    url = "https://example.com/accept-invite" // Redirect URL after accepting
+)
+```
+{% /multicode %}
+
+# Secure resources with team permissions
+
+Control access to documents and resources using team-based permissions. This ensures data isolation between tenants.
+
+{% multicode %}
+```client-web
+import { Client, Databases, ID, Permission, Role } from "appwrite";
+
+const client = new Client()
+    .setEndpoint('https://<REGION>.cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>');
+
+const databases = new Databases(client);
+
+// Create a document that only members of "Example Corp" tenant can access
+const document = await databases.createDocument(
+    'invoices_db',
+    'invoices',
+    ID.unique(),
+    {
+        title: 'Q2 Invoice',
+        amount: 2500.00,
+        customer: 'Example Customer',
+        status: 'pending'
```

Comment:
the tenant_id would need to be set here too.

Reply:
Done